src/lib/db/client.ts

```diff
@@ -64,8 +64,8 @@ export interface IDbInsert {
 
 export interface TableResult {
   result: any[],
-  fields: string[]
-  totalRecords: Number
```

Comment:
`totalRecords` not being a primitive caused trouble down the line and I saw no usage of the non-primitive `Number` type, so I changed it.